src/1-Utilities/utilities.ts

```diff
@@ -21,3 +21,23 @@ export const circleFilterUnique = (list:CircleListItem[]):CircleListItem[] =>
 
 //Converts underscores to spaces and capitalizes each word
 export const makeDisplayText = (text:string = ''):string => text.toLowerCase().split('_'||' ').map((s) => s.charAt(0).toUpperCase() + s.substring(1)).join(' ');
+
+export const makeAbbreviatedText = (text: string = '', abbreviateLastWord: boolean = true): string => 
+    makeDisplayText(text).split(' ').map((w, i, arr) => (i === arr.length - 1 && !abbreviateLastWord) ? w : w.charAt(0)).join(' ');
+
+/* Transform inputMap to Simple JavaScript Object */
+export const assembleRequestBody = (inputMap:Map<string,any>):Object => {
```

Comment:
On the portal, when I attempt to sign up a user, I get a ReferenceError stating that finalMap is not defined